content/docs/tasks/multicluster/gateways/index.md

```diff
@@ -139,14 +139,61 @@ running in a second cluster.
 
 ## Send remote cluster traffic using egress gateway
 
-If you want to route traffic from `cluster1` via a dedicated
-egress gateway, instead of directly from the sidecars,
+If you want to route traffic from `cluster1` via a dedicated egress gateway, instead of directly from the sidecars,
 use the following service entry for `httpbin.bar` instead of the one in the previous section.
 
 {{< tip >}}
 The egress gateway used in this configuration cannot also be used for other, non inter-cluster, egress traffic.
+
+If `$CLUSTER2_GW_ADDR` is an IP address, use the `$CLUSTER2_GW_ADDR - IP address` option.  If `$CLUSTER2_GW_ADDR` is a hostname, use the `$CLUSTER2_GW_ADDR - hostname` option.
```

Comment:
This should come after the tip, not inside it.

Reply:
Is there more to consider than that?
^^^ @rshriram

Reply:
Yeah. this is not a tip. this is essentially two slightly different service entries.